nixos/modules/services/databases/redis.nix

```diff
@@ -218,7 +187,7 @@ in
     };
 
     users.users.redis =
-      { name = cfg.user;
+      { name = "redis";
```

Comment:
`name` doesn't need to be set here. AFAIK, it's implied by `users.users.redis`.

Reply:
updated.